diff --git a/src/lib/import/importers/MercadoLivreImporter.ts b/src/lib/import/importers/MercadoLivreImporter.ts
index 9abfb2d..637ebca 100644
--- a/src/lib/import/importers/MercadoLivreImporter.ts
+++ b/src/lib/import/importers/MercadoLivreImporter.ts
@@ -157,8 +157,8 @@ export class MercadoLivreImporter implements Importer {
     if (itemId) {
       try {
         const [item, description] = await Promise.all([
-          fetchJson(`https://mercadolibre.com{itemId}`),
-          fetchJson(`https://mercadolibre.com{itemId}/description`).catch(() => null),
+          fetchJson(`https://api.mercadolibre.com/items/${itemId}`),
+          fetchJson(`https://api.mercadolibre.com/items/${itemId}/description`).catch(() => null),
         ]);
 
         return {
